selfdrive/car/toyota/interface.py

```diff
@@ -290,6 +290,16 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.6], [0.1]]
       ret.lateralTuning.pid.kf = 0.00006
 
+    elif candidate == CAR.ALPHARD_TSS2:
+      stop_and_go = True
+      ret.safetyParam = 73
+      ret.wheelbase = 3.00
+      ret.steerRatio = 14.2
+      tire_stiffness_factor = 0.444
+      ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.19], [0.02]]
+      ret.mass = 4305. * CV.LB_TO_KG + STD_CARGO_KG
+      ret.lateralTuning.pid.kf = 0.00007818594
+
```

Comment:
parameters other than wheelbase and mass were copied from CAR.SIENNA

Reply:
NOTE:  in the code below, there is an "Improved longitudinal tune" section.  Since jpM1 originally overwrote the RAV4H_TSS2 profile for his Alphard, he was hitting that code path.  But I don't think it was intentional, so I leave it as "Default longitudinal tune" unless someone makes an argument otherwise.